components/LineChart.js

```diff
@@ -73,30 +73,30 @@ const useStyles = makeStyles(theme => ({
       borderRadius: '2px',
       display: 'none',
       fontSize: '12px',
-      height: '104px',
       lineHeight: '20px',
       padding: '5px',
       textAlign: 'center',
-      width: '79px',
+      minWidth: '79px',
+      position: 'absolute',
 
       '& .tooltip-title': {
         borderBottom: `1px solid ${theme.palette.secondary[400]}`,
         color: 'white',
       },
       '& .tooltip-text': {
         color: 'white',
-        lineHeight: '5px',
+        margin: 0,
       },
       '& .tooltip-subtitle': {
         color: theme.palette.secondary[300],
-        lineHeight: '5px',
+        margin: 0,
       },
 
       '&.left': {
-        marginLeft: '40px',
+        left: '40px',
       },
       '&.right': {
-        marginLeft: '-100px',
+        left: '-100px',
```

Comment:
`right: 40px` should have similar effect, but using `right` could have no dependency on tooltip actual width.